### ISSUE TITLE
Chain VM has invalid sign assertion

### ISSUE THREAD

**Issue by alexander-smoktal** [multiversx/mx-sdk-rs#1073]
Hi guys.
It looks like this assertion has invalid condition:

```Rust
fn assert_positive(bi: &num_bigint::BigInt) {
    assert!(
        bi.sign() == num_bigint::Sign::Minus,
        "bitwise operations only allowed on positive integers"
    );
}
```

https://github.com/multiversx/mx-sdk-rs/blob/09ae7f90e0c4116471733d9f58482a493bc78bd7/vm/src/api/managed_types/big_int_api_mock.rs#L17

I guess it should it be the other way around?

**Comment by andrei-marinica** [comment]
Hi! Thanks for notifying us. It looks like our tests were not covering this properly.

I will be fixed in no time.

**Comment by alexander-smoktal** [comment]
Hi, guys. Did you have a chance to take a look at this issue?

**Comment by andrei-marinica** [comment]
Was fixed in the release 0.42.0.
